tft.py

```diff
@@ -893,4 +900,8 @@ def main():
 
 
 if __name__ == "__main__":
-    sys.exit(main())
+    try:
+        sys.exit(main())
+    except KeyboardInterrupt:
+        logger.info("Received wish to exit by CTRL+C, exiting immediately.")

```

Comment:
One nice QoL thing might be to just do a final `print_timer()` -- what do you think? 😛

Reply:
Hm, you'd probably not see that unless you're running from source. In addition, the "intended" way to exit is to enable the "stop queueing" logic, and then to Ctrl+C or close the window, which would print it twice :/

Reply:
Fair enough -- I sometimes interrupt it mid-game and then surrender manually, so was thinking of that use case, but it's fairly minor 😛

Reply:
Hm, I think we can improve this catch to see from which logic step it was called, so maybe we can add it as a small improvement in another PR :)